app/models/user.rb

```diff
@@ -40,7 +40,7 @@ class User < ActiveRecord::Base
 
   has_one :user_option, dependent: :destroy
   has_one :user_avatar, dependent: :destroy
-  has_one :primary_email, -> { where(primary: true)  }, class_name: 'UserEmail', dependent: :destroy, autosave: true
+  has_one :primary_email, -> { where(primary: true)  }, class_name: 'UserEmail', dependent: :destroy, autosave: true, validate: false
```

Comment:
Does it mean we can end up in a case where a user has emails but none of them are primary?

Reply:
There is another validation which ensures that `primary_email` is present, on line 109 below:

```
validates :primary_email, presence: true
```